src/main/java/edu/oregonstate/cs361/battleship/Coordinate.java

```diff
@@ -19,5 +19,30 @@ public int getAcross() {
         return Across;
     }
 
+    @Override
+    public int hashCode() {
+    	return Across * 10000 + Down;
+    }
+
+    @Override
```

Comment:
Upload new functional feature for Coordinate so that the code can run correctly.